# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/systemservices/objc_video_capture/video_capture_avfoundation.mm
# Commit: e85b278387fc
# Full Hash: e85b278387fc5259efead947f5069c9c2dcbffad
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2023-01-11 21:56:57
# Regressor Bug: 1806605
# File Overlap Count: 1
# Description:
#   Bug 1806605 - Pass TrackingId instead of nsCString to CaptureStage. r=padenot
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D165824
# ==============================================================================

diff -r 0fa64a630cc7 -r e85b278387fc dom/media/systemservices/objc_video_capture/video_capture_avfoundation.mm
--- a/dom/media/systemservices/objc_video_capture/video_capture_avfoundation.mm	Wed Jan 11 11:47:45 2023 +0000
+++ b/dom/media/systemservices/objc_video_capture/video_capture_avfoundation.mm	Wed Jan 11 11:47:45 2023 +0000
@@ -228,10 +228,10 @@
   return rv;
 }
 
-void VideoCaptureAvFoundation::SetTrackingId(const char* _Nonnull aTrackingId) {
+void VideoCaptureAvFoundation::SetTrackingId(uint32_t aTrackingIdProcId) {
   RTC_DCHECK_RUN_ON(&mChecker);
   MutexLock lock(&api_lock_);
-  mTrackingId = Some(nsCString(aTrackingId));
+  mTrackingId.emplace(TrackingId::Source::Camera, aTrackingIdProcId);
 }
 
 void VideoCaptureAvFoundation::StartFrameRecording(int32_t aWidth, int32_t aHeight) {